libs/utils/decorate.js

```diff
@@ -88,12 +88,13 @@ export function handleFocalpoint(pic, child, removeChild) {
     const childData = child.childNodes;
     if (removeChild) childData.forEach((c) => c.nodeType === Node.TEXT_NODE && c.remove());
   }
+  if (!text) return;
   const directions = text.trim().toLowerCase().split(',');
   const [x, y = ''] = directions;
   image.style.objectPosition = `${x} ${y}`;
 }
 
-export async function decorateBlockBg(block, node) {
+export async function decorateBlockBg(block, node, addHandleFocalpoint = false) {
```

Comment:
I know I initially suggested this, but let's change it to be an object instead of bool. Also nit change to be `useHandleFocalpoint`
```suggestion
export async function decorateBlockBg(block, node, { useHandleFocalpoint = false } = {}) {
```